source/administrator-guide/configuring-the-catalog/portal-configuration.rst

```diff
@@ -0,0 +1,142 @@
+.. _portal-configuration:
+
+Portal configuration
+####################
+
+From the ``admin console`` > ``settings`` user can configure portal under the ``sources`` section.
```

Comment:
I'd rephrase: Portals can be configured under the sources section in admin console > settings